.semaphore/semaphore.yml

```diff
@@ -28,7 +28,8 @@ blocks:
         commands:
           - cache restore bundle-gems-baseline
           # occasionally clear: `cache delete bundle-gems-baseline`
-          - cache restore bundle-gems-$(checksum Gemfile.lock)
+          - cache delete bundle-gems-$(checksum Gemfile.lock)
```

Comment:
you shouldn't ever need to do this.  just do `bundle update`

Reply:
Yeah, I just don't understand the whys and whens of these processes yet. I'm pretty sure I restored to master with this config.